src/createCycleMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import {run} from '@cycle/xstream-run';
+import actionDriver, { actionListener } from './drivers/actionDriver';
+import stateDriver, { stateListener } from './drivers/stateDriver';
```

Comment:
Maybe have the drivers do this:

```javascript
const action = actionDriver(store);
const { actionListener } = action;
drivers.ACTION = action.driver;
```

or something.

Or maybe still leave both of these drivers inlined, it wasn't super bad.